R/crc.R

```diff
@@ -38,34 +34,128 @@ crc <- function(data, freq.column, binary.variables, method = "poisson", corr.th
   corr_plot <- ggcorrplot::ggcorrplot(corr, type = "upper", lab = TRUE) +
     ggplot2::theme(legend.position = "none")
 
-  #Poisson Modeling ===================================
+  if(is.null(formula)){
```

Comment:
This checks to verify formula is null and then the nested if statements uses formula, wouldn't the nested if statements require it to be is.not.null(formula) to work?